basiclti/basiclti-util/pom.xml

```diff
@@ -44,6 +44,10 @@
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
 		</dependency>
+		<dependency>
+            <groupId>org.sakaiproject.kernel</groupId>
+            <artifactId>sakai-component-manager</artifactId>
+        </dependency>
         <dependency>
             <groupId>commons-logging</groupId>
             <artifactId>commons-logging</artifactId>
```

Comment:
Github is accurate. You are most likely using a different combination of spaces and tabs as the previous lines.